docs/concepts/storage/persistent-volumes.md

```diff
@@ -486,6 +489,24 @@ parameters:
   ```
   $ kubectl create secret generic heketi-secret --type="kubernetes.io/glusterfs" --from-literal=key='opensesame' --namespace=default
   ```
+  Example of a secret can be found in [glusterfs-provisioning-secret.yaml](glusterfs-provisioning-secret.yaml).
```

Comment:
`glusterfs-provisioning-secret.yaml` does not exist in that location. Can you add that yaml file?